### ISSUE TITLE
Feature request - use .gitignore file for ignore list

### ISSUE THREAD

**Issue by orrpan** [lukasz-wronski/vscode-ftp-sync#206]

**Comment by GXGOW** [comment]
I hope you'd want this in addition to the current implementation, because obviously there are files not on my GitHub repository which I want on my web server.

**Comment by orrpan** [comment]
@GXGOW  yes of course, just wanted the feature to have an additional option like 
`"ignorefile": ".file_using_gitignore_standard"`

**Comment by GXGOW** [comment]
@orrpan Oh no problem then. This could indeed be a good addition
